plugins/module_utils/fusion.py

```diff
@@ -100,39 +101,42 @@ def get_fusion(module):
 
     issuer_id = module.params[PARAM_ISSUER_ID]
     key_file = module.params[PARAM_PRIVATE_KEY_FILE]
+    access_token = module.params[PARAM_ACCESS_TOKEN]
 
     config = fusion.Configuration()
     config.host = environ.get(ENV_API_HOST, environ.get(ENV_HOST, config.host))
     config.token_endpoint = environ.get(ENV_TOKEN_ENDPOINT, config.token_endpoint)
-
-    if issuer_id is not None and key_file is not None:
-        try:
-            config.issuer_id = issuer_id
-            config.private_key_file = key_file
-            client = fusion.ApiClient(config)
-            client.set_default_header("User-Agent", user_agent)
-        except Exception:
-            module.fail_json(msg="Unknown failure. Please contact Pure Support")
+    
+    if access_token is not None:
+        config.access_token = access_token
+    elif issuer_id is not None and key_file is not None:
+        config.issuer_id = issuer_id
+        config.private_key_file = key_file
+    elif ENV_ACCESS_TOKEN in environ:
+        config.access_token = environ.get(ENV_ACCESS_TOKEN)
     elif (
         ENV_ISSUER_ID in environ or ENV_APP_ID in environ
     ) and ENV_PRIVATE_KEY_FILE in environ:
-        try:
-            config.issuer_id = environ.get(ENV_ISSUER_ID, environ.get(ENV_APP_ID))
-            config.private_key_file = environ.get(ENV_PRIVATE_KEY_FILE)
-            client = fusion.ApiClient(config)
-            client.set_default_header("User-Agent", user_agent)
-        except Exception:
-            module.fail_json(msg="Unknown failure. Please contact Pure Support")
+        config.issuer_id = environ.get(ENV_ISSUER_ID, environ.get(ENV_APP_ID))
+        config.private_key_file = environ.get(ENV_PRIVATE_KEY_FILE)
     else:
         module.fail_json(
-            msg=f"You must set {ENV_ISSUER_ID} and f{ENV_PRIVATE_KEY_FILE} environment variables "
-            f"or the {PARAM_ISSUER_ID} and f{PARAM_PRIVATE_KEY_FILE} module arguments"
+            msg=f"You must set either {ENV_ISSUER_ID} and f{ENV_PRIVATE_KEY_FILE} or {ENV_ACCESS_TOKEN} environment variables. "
+            f"Or module arguments either {PARAM_ISSUER_ID} and f{PARAM_PRIVATE_KEY_FILE} or {ENV_ACCESS_TOKEN}"
```

Comment:
```suggestion
            f"Or module arguments either {PARAM_ISSUER_ID} and f{PARAM_PRIVATE_KEY_FILE} or {PARAM_ACCESS_TOKEN}"
```
Here you have wrong constant.